src/js/pages/services/task-details/TaskDetailsTab.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+
+import DescriptionList from '../../../components/DescriptionList';
+import MarathonTaskDetailsList from '../../../components/MarathonTaskDetailsList';
+import MesosStateStore from '../../../stores/MesosStateStore';
+import MesosSummaryStore from '../../../stores/MesosSummaryStore';
+import ResourcesUtil from '../../../utils/ResourcesUtil';
+import TaskDirectoryStore from '../../../stores/TaskDirectoryStore';
+import Units from '../../../utils/Units';
+
+class TaskDetailsTab extends React.Component {
+  getMesosTaskDetailsDescriptionList(mesosTask) {
+    if (mesosTask == null || !MesosSummaryStore.get('statesProcessed')) {
+      return null;
+    }
+
+    let headerValueMapping = {'ID': mesosTask.id};
+
+    let services = MesosSummaryStore.get('states')
+      .lastSuccessful()
+      .getServiceList();
+    let service = services.filter({ids: [mesosTask.framework_id]}).last();
+
+    if (service != null) {
+      headerValueMapping['Service'] = `${service.name} (${service.id})`;
+    }
+
+    let node = MesosStateStore.getNodeFromID(mesosTask.slave_id);
+
+    if (node != null) {
+      headerValueMapping['Node'] = `${node.hostname} (${node.id})`;
+    }
+
+    let sandBoxPath = TaskDirectoryStore.get('sandBoxPath');
+    if (sandBoxPath) {
```

Comment:
Sometimes have whitespace before/after a `let` or `if` statement in this method. Maybe you could pick one way and make sure spacing is consistent in the method :)